config/application.rb

```diff
@@ -110,9 +108,6 @@ def config.database_configuration
     config.autoloader = :zeitwerk
 
     # Custom directories with classes and modules you want to be autoloadable.
-    config.autoload_paths += Dir["#{config.root}/app"]
-    config.autoload_paths += Dir["#{config.root}/app/jobs"]
-    config.autoload_paths += Dir["#{config.root}/app/serializers"]
```

Comment:
These lines are redundant. Rails by default adds any folders under `app` into the autoload path.